.gitignore

```diff
@@ -26,7 +26,7 @@ junit.xml
 .vscode
 
 # docs
-docs
+/docs
```

Comment:
We now have a module called 'docs', so we have to limit the scope of this ignore.